ci: add luacheck linter (#23)
tarantool/metrics-export-role: README.md

cd metrics-export-role
```

> After that you need to install dependencies (`tt` is required).
> It's possible to install requirements only for running tests
> or linter check. To install all of this, run:
> 
> ```shell
> make deps
> ```
> 
> To install only tests requirements, run:
> 
> ```shell
> make deps depname=test
> ```
> 
> To install tests with coverage requirements, run:

Reviewer: ````suggestion
After that you need to install dependencies (`tt` is required).
To do it, run:

```shell
make deps
```
It's possible to install requirements only for running tests
or linter check. To install only tests requirements, run:

```shell
make deps depname=test
```

To install tests with coverage requirements, run:
````

Author: Edited.

cd metrics-export-role
```

After that you need to install dependencies (`tt` is required).
It's possible to install requirements only for running tests
or linter check. To install tests requirements, run:

```shell
make deps depname=test
```

To install tests with coverage requirements, run:

```shell
make deps depname=coverage
```

To install linter checks, run:

```shell
make deps depname=lint

```

> Finally to install all of this, simply run:

Reviewer: It's better to start with this one

Author: Edited.
